Dev6/Dev6/CommandFactory.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+
+namespace Dev6
+{
+    /// <summary>
+    /// Class for creates commands
+    /// </summary>
+    class CommandFactory
+    {
+        public CarCollection Catalog { get; private set; }
+        private ICommand Command { get; set; }
+
+        /// <summary>
+        /// Constructor initializes fields
+        /// </summary>
+        /// <param name="carCollection">Catalog of cars</param>
+        public CommandFactory(CarCollection carCollection) => Catalog = carCollection;
+
+        /// <summary>
+        /// Declare an enumeration
+        /// </summary>
+        public enum CatalogCommand
+        {
+            CountTypes = 1,
+            CountAll,
+            AveragePrice,
+            AveragePriceType,
+            Exit
+        }
+
+        /// <summary>
+        /// Gets command
+        /// </summary>
+        public void GetCommand()
```

Comment:
please use single responsibility. This method prompts the user to enter data, checks it and only then selects the command.  So split this method